t/errlog.t

```diff
@@ -932,3 +932,65 @@ maybe log lines: #1
 end
 \z
 --- skip_nginx: 2: <1.11.2
+
+
+
+=== TEST 23: the system default filter level is "debug"
```

Comment:
`init_by_lua*` is special, https://github.com/openresty/lua-nginx-module/blob/master/src/ngx_http_lua_phase.c#L33

I think we'd better add a test case. @spacewander